src/main/java/dev/openfga/sdk/api/client/ClientWriteRequest.java

```diff
@@ -15,18 +15,26 @@
 import java.util.List;
 
 public class ClientWriteRequest {
-    private List<ClientTupleKey> writes;
+    private List<ClientTupleKeyWithCondition> writes;
```

Comment:
Can we change these to `ClientTupleKey` and `ClientTupleKeyWithoutCondition` to be consistent with the API (though I like this naming better)

Reply:
Fixed these names in [17e8f04](https://github.com/openfga/java-sdk/pull/33/commits/17e8f04151c4e9c6fd96284c2b6b55df9e27622a)